Source/drlg_l2.cpp

```diff
@@ -1,4 +1,7 @@
 #ifndef SPAWN
+
+#include <algorithm>

```

Comment:
oh, boy. the amiga cross tools won't like this. it requires ```#include <cstdio>``` before algorithm.

Reply:
Does amiga not have min/max?

Reply:
Why does `cstdio` have anything to do with `algorithm`?
`algorithm` shouldn't be concerned with io at all:
https://en.cppreference.com/w/cpp/header/algorithm

Reply:
I noticed it earlier in another file I merged in from master. I had to prepend cstdio before to suppress the error. We'll see, I'll notice soon enough.

Reply:
https://github.com/AmigaPorts/devilutionX/commit/8e697cf7677cb61e9f2e3d2cd85f8973293c0cdc

Reply:
Probably broken headers or something. *shrug*

Reply:
```In file included from /opt/m68k-amigaos-20190207/lib/gcc/m68k-amigaos/6.5.0b/include/c++/cstdlib:75:0,
                 from /opt/m68k-amigaos-20190207/lib/gcc/m68k-amigaos/6.5.0b/include/c++/bits/stl_algo.h:59,
                 from /opt/m68k-amigaos-20190207/lib/gcc/m68k-amigaos/6.5.0b/include/c++/algorithm:62,
                 from /home/marlon/tmp/Programming/Amiga/Projects/devilutionX/Source/drlg_l3.cpp:3:
/opt/m68k-amigaos-20190207/m68k-amigaos/sys-include/stdlib.h:108:64: error: '_mbstate_t' has not been declared
 __stdargs int _mblen_r (struct _reent *, const char *, size_t, _mbstate_t *);```

Reply:
According to http://eab.abime.net/showthread.php?p=1314794 it just needs `#include <wchar.h>`. Should be fixed in the toolchain though.

Reply:
Yes, most likely. I googled the issue and that's what it threw at me. And it worked. 
Thanks, will notify the maintainer of the toolchain.

Reply:
https://github.com/bebbo/amiga-gcc/issues/132